modules/react-native-status/desktop/CMakeLists.txt

```diff
@@ -26,12 +26,22 @@ include_directories(${StatusGo_INCLUDE_DIR})
 
 set(CONFIGURE_SCRIPT build-status-go.sh)
 
-file (STRINGS "../../../STATUS_GO_VERSION" STATUS_GO_VERSION)
+file(STRINGS "../../../STATUS_GO_VERSION" STATUS_GO_VERSION)
+
+# custom status-go builds are identified by '{date}-{time}-{commit}'
+if ("${STATUS_GO_VERSION}" MATCHES "[0-9][0-9][0-9][0-9][0-9][0-9]-[0-9][0-9][0-9][0-9][0-9][0-9]-([0-9a-zA-Z]+)")
```

Comment:
`[0-9]{6}-[0-9][0-9][0-9][0-9][0-9]{6}` is too difficult for whatever executes this file?

Reply:
Yup, I thought the same thing, but looking at the CMake documentation it is not supported.

Reply:
Yeah, it's really stupid:
https://cmake.org/cmake/help/v3.0/command/string.html
```
^         Matches at beginning of input
$         Matches at end of input
.         Matches any single character
[ ]       Matches any character(s) inside the brackets
[^ ]      Matches any character(s) not inside the brackets
 -        Inside brackets, specifies an inclusive range between
          characters on either side e.g. [a-f] is [abcdef]
          To match a literal - using brackets, make it the first
          or the last character e.g. [+*/-] matches basic
          mathematical operators.
*         Matches preceding pattern zero or more times
+         Matches preceding pattern one or more times
?         Matches preceding pattern zero or once only
|         Matches a pattern on either side of the |
()        Saves a matched subexpression, which can be referenced
          in the REGEX REPLACE operation. Additionally it is saved
          by all regular expression-related commands, including
          e.g. if( MATCHES ), in the variables CMAKE_MATCH_(0..9).
```